internals/plan/plan.go

```diff
@@ -317,6 +347,40 @@ const (
 	ActionIgnore   ServiceAction = "ignore"
 )
 
+// LogTargetsReplace represents a possible value for a service's `^log-targets`
+// field - either an explicit list of targets, or the special keyword "default".
+type LogTargetsReplace struct {
+	Targets []string
+	Default bool
```

Comment:
IMO these fields shouldn't be exported - makes testing a little harder though. Maybe the `LogTargetsReplace` struct shouldn't even be exported?

Reply:
I think it's probably fine as is. It keeps the plan structs simple and consistent. And besides, this is in "internals" anyway, not something that should be imported elsewhere.